app/helpers/script_helper.rb

```diff
@@ -7,7 +7,12 @@ def javascript_include_tag_without_preload(...)
   end
 
   def javascript_packs_tag_once(*names, **attributes)
-    @scripts = @scripts.to_h.merge(names.index_with(attributes))
+    scripts = RequestStore.store[:scripts]
```

Comment:
request_store is only a transitive dependency of lograge, if we're using it directly like this, WDYT of making it an explicit dependency?

Reply:
Oops, yes, meant to do that

Reply:
Added in 6970689f20